zip-0000.rst

```diff
@@ -512,12 +577,14 @@ ZIP Status Field
   once rough consensus is reached in PR/forum posts from Draft Process ZIP.
 
 * Proposed: Typically the stage after Draft, added to a ZIP after
-  consideration, feedback, and rough consensus from the community. The ZIP
-  Editors must validate this change before it is approved.
+  consideration, feedback, and rough consensus from the community.
+  The Proposed status is approved by consensus among the current ZIP
+  Editors.
 
 * Rejected: The status when progress hasn't been made on the ZIP in one
   year. Can revert back to Draft/Proposed if the Owner resumes work
```

Comment:
```suggestion
* Rejected: If no progress on a Draft or Proposed ZIP has been made for
  one year, the ZIP Editors SHOULD move it to Rejected status. It can revert
  back to Draft/Proposed if the Owner resumes work or resolves issues
```